After a_time_tm.c r1.22, times without seconds are no longer accepted
by the ASN.1 parser. This corresponds to RFC 5280, but not to ASN.1.

Index: test/openssl/test_asn1.rb
--- test/openssl/test_asn1.rb.orig
+++ test/openssl/test_asn1.rb
@@ -405,7 +405,7 @@ class  OpenSSL::TestASN1 < OpenSSL::TestCase
     encode_decode_test B(%w{ 17 0D }) + "160908234339Z".b,
       OpenSSL::ASN1::UTCTime.new(Time.utc(2016, 9, 8, 23, 43, 39))
     # Seconds is omitted
-    decode_test B(%w{ 17 0B }) + "1609082343Z".b,
+    decode_test B(%w{ 17 0D }) + "160908234300Z".b,
       OpenSSL::ASN1::UTCTime.new(Time.utc(2016, 9, 8, 23, 43, 0))
     begin
       # possible range of UTCTime is 1969-2068 currently
@@ -432,7 +432,7 @@ class  OpenSSL::TestASN1 < OpenSSL::TestCase
       OpenSSL::ASN1::GeneralizedTime.new(Time.utc(2016, 12, 8, 19, 34, 29))
     encode_decode_test B(%w{ 18 0F }) + "99990908234339Z".b,
       OpenSSL::ASN1::GeneralizedTime.new(Time.utc(9999, 9, 8, 23, 43, 39))
-    decode_test B(%w{ 18 0D }) + "201612081934Z".b,
+    decode_test B(%w{ 18 0F }) + "20161208193400Z".b,
       OpenSSL::ASN1::GeneralizedTime.new(Time.utc(2016, 12, 8, 19, 34, 0))
     # not implemented
     # decode_test B(%w{ 18 13 }) + "20161208193439+0930".b,
